webapp/channels/src/packages/mattermost-redux/src/action_types/hosted_customer.ts

```diff
@@ -6,15 +6,5 @@ import keyMirror from 'mattermost-redux/utils/key_mirror';
 export default keyMirror({
     SELF_HOSTED_PRODUCTS_REQUEST: null,
```

Comment:
Do we still need this self hosted get products ?

Reply:
Yes, for the pricing modal